### PR TITLE
Typeset with SILE

### PR DESCRIPTION
**Pull request by simoncozens**

### CODE REVIEW

**Comment by khaledhosny** [comment]
Funny, because typesetting this text with SILE was the first item on my list for this morning :) (I was considering also adding `jalt` feature to the font and figuring how to use that obscure SILE feature that would let me use glyph alternates for justification).

**Comment by simoncozens** [comment]
screen shot 2016-10-28 at 13 31 48

**Comment by khaledhosny** [comment]
Hmm, we want the chapter to be a single paragraph.

**Comment by simoncozens** [comment]
That leads to some interesting layout bugs with the end-of-aya glyphs:
screen shot 2016-10-28 at 13 37 07

**Comment by khaledhosny** [comment]
Rather the bidi is failing, looks like it faild to re-reverse the runs. The glyphs are shaped correctly but everything is now output from left to right.

**Comment by simoncozens** [comment]
Yet another bug in the pushback mechanism. :(

Using `\disable-pushback` it looks like this:
screen shot 2016-10-28 at 13 46 50

The whole function needs rethinking. :-(

File: sile/classes/quran.lua [hunk 1]
```diff
@@ -183,7 +183,6 @@ SILE.registerCommand("aya", function (o,c)
     end)
   end)
   SILE.typesetter:typeset(o.text)
-  SILE.call("par") --- ?
```
Comment (khaledhosny) [inline sile/classes/quran.lua]: There should be a space in place of the `\par`. I tried `SILE.typesetter:typeset(o.text.." ")` and seems to be fine.

**Comment by khaledhosny** [comment]
Where are page numbers are formatted? I want to give converting from European to Arabic-Indic digits a try.

**Comment by simoncozens** [comment]
Look at `SILE.formatCounter` in `counters.lua`

**Pull request merged**